5.sc_Morphology_Logistic_Regression/1.train_models/0.train_single_class.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        aggregation = ast.literal_eval(config["logistic_regression_params"]["aggregation"])
</pre><p>Didn't know this was possible, cool</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>